.github/workflows/update_references.yml

```diff
@@ -0,0 +1,47 @@
+name: Receive repository dispatch event
+
+on:
+  # Listen to a repository dispatch event by the name of `dispatch-event`
+  repository_dispatch:
+    types: [update-references]
+  workflow_dispatch:
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  create-pull-request:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+
+      - name: Setup Node.js 20
+        uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version: 20.x
+
+      - name: Set github commit user
+        env:
+          GITHUB_EMAIL: ${{ vars.GH_EMAIL }}
+          GITHUB_USER: ${{ vars.GH_USER }}
+        run: |
+          git config --global user.email $GITHUB_EMAIL
+          git config --global user.name $GITHUB_USER
+
+        # Set branch name to be used as environment variable
+      - name: Set Branch Name
+        run: echo "BRANCH_NAME=$(echo $(date +%s))" >> $GITHUB_ENV
+
+        # Create new branch, download, and commit changes to the new branch
+      - name: Create new branch
+        run: |
+          git checkout -b ${{ env.BRANCH_NAME }}
+          curl -L -o ${{ vars.REF_LOC }} ${{ vars.REMOTE_REF }}
```

Comment:
Same question here, should we be saving these variables in the github environment variables?

Reply:
Also what does `vars.REF_LOC` and `vars.REMOTE_REF` represent? We can chat offline about it